docs/getting-started/generated-app.md

```diff
@@ -7,24 +7,28 @@ slug: /getting-started
 
 # The Generated Service
 
-Amplication auto-generates fully functional human-readable and editable services based on TypeScript and Node.js.
+When you use Amplication to create your backend service, we automatically generate a codebase that follows best practices for software development and is fully functional, human-readable, and editable.
 
-The generated services include NestJS, Prisma, REST API, GraphQL API, a React admin UI, authentication, authorization, logging, and more.
```

Comment:
Maybe we can mention these capabilities in the key-features list detailed below?

Reply:
Hey @PazYanoverr , a lot of these technologies are already listed, except for what we use for logging. I'm not sure what we use for this one.